Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3_test.py

```diff
@@ -278,6 +278,18 @@ def test_domain_command(mocker, requests_mock):
     assert results[0].execution_metrics is None
     assert results[0].outputs == expected_results
 
+    mock_response = {'error': {'code': 'NotFoundError'}}
```

Comment:
can you update the given-when-then docs?

Reply:
Yes. Which are these docs? It's the first time I have been asked to update them.

Reply:
doc strings for the test function, you can check crowdstrikefalcon_test.py for reference.